src/components/text-input/TextInput.scss

```diff
@@ -1,55 +1,105 @@
 @import '../../styles/variables';
+@import '../../styles/inputs';
 
-/**************************************
- * Text Input
- **************************************/
+.bdl-TextInput-input {
+    @include common-typography;
+
+    height: 32px;
+    margin-top: 5px;
+
+    &[type="text"],
+    &[type="password"],
+    &[type="search"],
+    &[type="email"],
+    &[type="url"],
+    &[type="tel"],
+    &[type="number"] {
+        @include box-inputs;
+
+        width: 262px;
+
+        &:hover {
+            @include box-inputs-hover;
```

Comment:
@includes will be coming from _textInputs.scss, so these will be text-input-styles, text-input-hover-styles etc.